staff/maryam-malek/cv/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+   
+</head>
+<body>
+        <header>
+            <h1 id="name">Richard House</h1>
+            <address>
+                Email adress: <br> 
+                <a href="mailto:webmaster@example.com">webmaster@example.com</a><br> 
+                Portafolio:<br>
+                <a href="mailto:webmaster@example.com">example.com</a>.<br>
+                Street beta<br>
+                London
+            </address>
+        </header>  
+        <section id="studies">
+            <h2>Studies</h2> 
+                <article>
+                    <h4> Compulsory High School Education</h4>
+                    <p>Oxford High School</p>
+                    <p>London</p>
+                </article>
+                <article>
+                    <h4>Arts and Humanities</h4>
+                    <p>University of London</p>
+                    <p>London</p>
+                </article>
+                <article>
+                    <h4>Master in Roman art</h4>
+                    <p>University of London</p>
+                    <p>London</p>
+                </article>           
+        </section>
+     <section id="experience">
+        <h2>Experience</h2> 
+        <article>
+                <h4> Lorem ipsum dolor sit amet eos?</h4>
+                <p>ipsum dolor sit amet eos</p>
+                <p>London</p>
+            </article>
```

Comment:
the indentation is confusing here

Reply:
Nice. You can use ASIDE and after diferents SECTION.

https://www.w3schools.com/tags/tag_aside.asp

Reply:
`aside` should be used for aside not content, but not for main content. in this case you seem to present main content, so please, understand the meaning and use of  `aside`, correct this html. just RTFM https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside